### DISCUSSION TITLE
Change the MasterWindowFlagsFloating dynamically

### DISCUSSION THREAD

**Discussion by mtp** [AllenDang/giu#104] [Q&A] [answered]
Hello,

I'm trying to change the MasterWindowFlagsFloating dynamically.

To do so, I create a dedicated function, which I'm calling from the main loop. But it's not working. Do you know if it's possible to change the on top flag dynamically?

`func switchOnTopState() {
	log.Println("UI : Switching On Top State")
	if onTopState {
		glfw.WindowHint(glfw.Floating, glfw.False)
		onTopState = false
	} else {
		glfw.WindowHint(glfw.Floating, glfw.True)
		onTopState = true
	}
	g.Update()
}`

Thanks for your help

Update : 
As read there (https://github.com/go-gl/glfw/blob/master/v3.3/glfw/glfw.go) it seems that Hints cannot be modified once the glfw is initialized

> The values you set hints to are never reset by GLFW, but they only take
> effect during initialization. Once GLFW has been initialized, any values you
> set will be ignored until the library is terminated and initialized again.

**Comment by AllenDang** [comment]
Yes, glfw rules the world...

**Comment by mtp** [comment, marked as answer]
Thanks for your answer Allen.

OK, so after some research on github, it have been addressed by this issue :
https://github.com/glfw/glfw/issues/43

So we only have to use SetAttrib function from window.go from the go glfw, it works perfectly !
